plugins/tech-insights-node/api-report.md

```diff
@@ -37,6 +38,9 @@ export interface FactCheckerFactory<
   ): FactChecker<CheckType, CheckResultType>;
 }
 
+// @public
+export type FactLifecycle = TTL | ITL;
```

Comment:
I don't think the amount of items matches the definition of _time_ to live. I think I'll change the naming of these to spell out the actual meaning for clarity.

Reply:
I'd say it does for IP for example where it's a discrete count, although counting hops rather than items. Still share the concern that it's not very clear though. Maybe just a simple `maxItems` or such instead of `itl`/`itemsToLive`?

Reply:
Was thinking of `items` actually, but `maxItems` is probably better 👍